providers/provider_default.go

```diff
@@ -110,17 +110,34 @@ func (p *ProviderData) GetEmailAddress(s *sessions.SessionState) (string, error)
 	return "", errors.New("not implemented")
 }
 
+func (p *ProviderData) GetUserDetails(s *sessions.SessionState) (*UserDetails, error) {
+	return nil, NewNotImplementedError("")
+}
+
 // GetUserName returns the Account username
 func (p *ProviderData) GetUserName(s *sessions.SessionState) (string, error) {
 	return "", errors.New("not implemented")
 }
 
+func (p *ProviderData) GetGroups(s *sessions.SessionState, f string) (map[string]string, error) {
+	return map[string]string{}, errors.New("not implemented")
+}
+
 // ValidateGroup validates that the provided email exists in the configured provider
 // email group(s).
 func (p *ProviderData) ValidateGroup(email string) bool {
 	return true
 }
 
+// ValidateExemptions checks if we can allow user login dispite group membership returned failure
```

Comment:
Small typo in the comment
```suggestion
// ValidateExemptions checks if we can allow user login despite group membership returned failure
```